modules/conversion-center/assets/scss/widgets/link-in-bio-base.scss

```diff
@@ -76,42 +81,55 @@
 		--e-link-in-bio-border-style: solid;
 	}
 
-	@mixin full-height($breakpoint) {
-		@media (max-width: $breakpoint) {
-			--e-link-in-bio-container-height: 100vh;
+	@mixin full-height($min: null, $max: null) {
+		$query: null;
 
-			@supports (height: 100dvh) {
-				--e-link-in-bio-container-height: 100dvh;
-			}
+		@if $min != null and $max != null {
+			$query: "(min-width: " + $min + ") and (max-width: " + $max + ")";
+		} @else if $max != null {
+			$query: "(max-width: " + $max + ")";
+		} @else if $min != null {
+			$query: "(min-width: " + $min + ")";
+		}
+
+		@if $query != null {
+			@media #{$query} {
```

Comment:
Will we generated a media query for each and every breakpoint?
We always try to avoid that as much as possible.
Can you please share a link or screenshot of the design and of the controls?

Reply:
@hein-obox this will setup only the appropriate media query based on the passed params.  $query being the building of those params into the media query. 

It is tied to the .is-full-height-[breakpoint] utility classes below. The requirement here is for full height to be selectable by the author for each breakpoint they have configured. Therefore we need to provide the appropriate mappings. This was called out in a previous code review and eventually passed. Here we have simply extended the existing pattern to ensure media queries apply only to the specified breakpoint rather than building on each of in a mobile or desktop first pattern.

See screenshot
![Screenshot 2024-05-21 at 11 22 15](https://github.com/elementor/elementor/assets/1838952/ceee1c66-bc1e-4e28-9a05-6369ab54f13e)